src/pages/index.tsx

```
@@ -1,55 +1,47 @@
-import dayjs from "dayjs";
-import { useState } from "react";
+import ErrorBoundary from "@/components/commons/ErrorBoundary";
+import Skeleton from "@/components/commons/Skeleton";
+import MyReservation from "@/components/pages/dashboard/components/MyReservation";
+import { useGetUserReservation } from "@/components/pages/dashboard/hooks/useGetUserReservation";
+import Error from "@/components/pages/seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)